src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--29">
```

Comment:
Use a modifier for month-length (e.g., calendar--29) instead of hardcoding dates.